app/scripts/views/mixins/flow-begin-mixin.js

```diff
@@ -10,10 +10,7 @@ define(function (require, exports, module) {
 
   module.exports = {
     afterRender () {
-      const flowId = this.flow.get('flowId');
-      const flowBegin = this.flow.get('flowBegin');
-
-      this.metrics.logFlowBegin(flowId, flowBegin);
+      this.logFlowEventOnce('begin');
```

Comment:
I was just looking at the function signature of `logFlowEventOnce` - does it matter that [no viewName](https://github.com/mozilla/fxa-content-server/blob/3495a24685f4dcd95fe3c6d82c12eb8a5bb5763d/app/scripts/views/base.js#L727) is passed in? Should the `viewName` default to `this.viewName` in `logFlowEventOnce` and `logFlowEvent`?